### PR TITLE
Fix typos from switch to Python 3

### PR DESCRIPTION
**Pull request by hauk142**
Removed eval() from eval(input()).

### CODE REVIEW

File: docs/exploring-gnuradio/dial_tone.py [hunk 1]
```diff
@@ -41,5 +41,5 @@ def build_graph():
 if __name__ == '__main__':
     tb = build_graph()
     tb.start()
-    eval(input('Press Enter to quit: '))
+    input('Press Enter to quit: ')
```
Comment (mbr0wn) [inline docs/exploring-gnuradio/dial_tone.py]: In Python 2, this doesn't work anymore.

Comment (mbr0wn) [inline docs/exploring-gnuradio/dial_tone.py]: Unless, of course, if you `from builtins import input`. In that case though, you added 'future' as a dependency for GNU Radio on Py2k.

**Comment by hauk142** [comment]
Yes of course, you're absolutely right. :)

Could do something like this:
`try: input = raw_input
except NameError: pass`

**Comment by mbr0wn** [comment]
...but then it's a nop on Py2k :)

File: docs/exploring-gnuradio/dial_tone.py (hunk 1 above)
Comment (jmcorgan) [reply docs/exploring-gnuradio/dial_tone.py]: I think this is okay for 3.8.

**Comment by jmcorgan** [comment]
I'm ok with having this left as 'input()' but the future module import needs to be added for py2 operation.

**Comment by hauk142** [comment]
Could also use input() from six

**Comment by jmcorgan** [comment]
That's an even better idea.

**Comment by mbr0wn** [comment]
Note that `six` is also a new dependency (if it's not already one). It's not part of the vanilla Python libs.

**Comment by hauk142** [comment]
`six` is already used in several modules :)

**Comment by mbr0wn** [comment]
Argh, that's even worse, it means we have an untracked dependency. @jmcorgan do you know what to do to add Python dependencies? `six` needs tracking.

**Comment by jmcorgan** [comment]
I thought it already was. The entire python3 port depends on it.

**Comment by skoslowski** [comment]
six is listed in the "Release 3.8 Dependency Analysis" document

**Comment by mbr0wn** [comment]
OK good, false alarm them. I tried figuring out which CMake build step checks for six, couldn't find it with in a few seconds.

**Pull request merged**